cli/tests/e2e/snapshots/test_ci/test_github_ci_bad_base_sha/results.txt

```diff
@@ -111,15 +111,6 @@ Running semgrep-core with command:
       "pattern": "$X == 4",
       "severity": "ERROR"
     },
-    {
```

Comment:
Could you explain this?

Reply:
yeah I'm just hoping this is CI being weird and nondeterministic